src/bracuganda/res/values/strings.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 	<string name="app_name">BRAC</string>
-	<string name="fixed_app_url">https://brac-ug.app.medicmobile.org</string>
+	<string name="scheme">https</string>
+	<string name="fixed_app_url">brac-ug.app.medicmobile.org</string>
```

Comment:
It's not technically a URL any more - can we rename this variable to `app_host` or something?

Reply:
`app_host` it is!